Mathlib/CategoryTheory/Subobject/Basic.lean

```diff
@@ -236,6 +239,10 @@ theorem mk_arrow (P : Subobject X) : mk P.arrow = P :=
     obtain ⟨e⟩ := @Quotient.mk_out' _ (isIsomorphicSetoid _) Q
     exact Quotient.sound' ⟨MonoOver.isoMk (Iso.refl _) ≪≫ e⟩
 
+lemma mk_surjective {X : C} (S : Subobject X) :
+    ∃ (A : C) (i : A ⟶ X) (_ : Mono i), S = Subobject.mk i :=
+  ⟨_, S.arrow, inferInstance, by simp⟩
```

Comment:
```suggestion
  ⟨_, S.arrow, inferInstance, (mk_arrow _).symm⟩
```

Reply:
When a fast `simp` works, I do not think there is need to provide a term proof.